### PR TITLE
create .babelrc as per required

### PR DESCRIPTION
**Pull request by raitalharehman**
This file is required for storybook

### CODE REVIEW

File: src/storybook/.babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+   "presets": ["react"]
```
Comment (piotrgrundas) [inline src/storybook/.babelrc]: Could we use the same as in the saleor:
`["@babel/preset-env", "@babel/preset-react"]`?

Comment (raitalharehman) [reply src/storybook/.babelrc]: we can use `["babel-preset-env", "babel-preset-react"]` cause saleor storefront using these packages.

Comment (raitalharehman) [reply src/storybook/.babelrc]: @piotrgrundas can we use? ["babel-preset-env", "babel-preset-react"]

Comment (piotrgrundas) [reply src/storybook/.babelrc]: Yes, that was my suggestion.

Comment (raitalharehman) [reply src/storybook/.babelrc]: @piotrgrundas  Done

**Pull request merged**